README.md

```diff
@@ -126,6 +189,76 @@ client.on('connected', async (conn) => {
 client.start();
 ```
 
+# PulseQueue
+
+A PulseQueue declares a queue and listens for messages on that
+queue, invoking a callback for each messages.
+
+The options to the constructor are:
+
+```javascript
+let pq = new pulse.PulseQueue({
+  client,                // Client object for connecting to the server
+  bindings: [{           // exchange/routingKey patterns to bind to
+    exchange,            // Exchange to bind
+    routingKeyPattern,   // Routing key as string
+    routingKeyReference, // Reference used to parse routing keys (optional)
+  }, ..],
+  handleMessage,         // handler for incoming messages
```

Comment:
Optional, right?

Reply:
No, what would we do with messages if not listening for them?